cloudformation/ebs-substack.cfn.json

```diff
@@ -131,24 +131,54 @@
                 }
               ]
             },
-            "io1"
+            "gp3"
           ]
         },
         {
-          "Fn::Equals": [
+          "Fn::Or": [
```

Comment:
I think you can use a single Or with 3 conditions in it. Same in all other places

Reply:
Done!